executor-scripts/linux/link

```diff
@@ -80,7 +80,7 @@ create)
 		${MOCK} ip link add link "${IF_VLAN_RAW_DEVICE}" name "${IFACE}" type vlan id "${IF_VLAN_ID}"
 	fi
 	;;
-up)
+pre-up)
```

Comment:
Need to find a better solution for this. The problem here is that, without this change, the link is only up'ed in the `up` phase. However, I want executors in the `up` phase to be executed *after* the link was up'ed successfully (as detected via netlink).